pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -87,8 +88,8 @@ type SchedulingQueue interface {
 }
 
 // NewSchedulingQueue initializes a priority queue as a new scheduling queue.
-func NewSchedulingQueue(stop <-chan struct{}) SchedulingQueue {
-	return NewPriorityQueue(stop)
+func NewSchedulingQueue(stop <-chan struct{}, lessFunc framework.LessFunc) SchedulingQueue {
```

Comment:
I think it is better to pass the `Framework` to the constructor, instead of just the `LessFunc`. We may want to use more functionalities of the framework in the scheduling queue in the future.